packages/components/select/tests/multi-select.test.tsx

```diff
@@ -0,0 +1,398 @@
+      const { user } = render(
+        <MultiSelect
+          items={[
+            {
+              label: "Numbers",
+              items: [{ label: "One", value: "one" }],
+            },
+            {
+              label: "Two",
+              value: "two",
+            },
+          ]}
+        />,
+      )
+
+      const input = await screen.findByRole("combobox")
+      expect(input).toBeInTheDocument()
+
+      await user.click(input)
+
+      const select = await screen.findByRole("listbox")
+      expect(select).toBeInTheDocument()
+      expect(select).toHaveTextContent(/numbers/i)
+      expect(select).toHaveTextContent(/one/i)
+    })
+
+    test("when items clicked twice, renders correctly", async () => {
+      const { user } = render(
+        <MultiSelect>
+          <Option value="one">One</Option>
+          <Option value="two">Two</Option>
+          <Option value="three">Three</Option>
+        </MultiSelect>,
+      )
+
+      const input = await screen.findByRole("combobox")
+
+      await user.click(input)
+
+      const option1 = await screen.findByRole("option", { name: /one/i })
+      await user.click(option1)
+
+      const option2 = await screen.findByRole("option", { name: /two/i })
+      await user.click(option2)
+
+      expect(option1).toBeVisible()
+      expect(option2).toBeVisible()
+    })
+  })
+
+  describe("click event", () => {
+    const ITEMS = [
+      {
+        label: "option1",
+        value: "option1",
+      },
+      {
+        label: "option2",
+        value: "option2",
+      },
+      {
+        label: "option3",
+        value: "option3",
+      },
+    ]
+
+    test("click option should work correctly", async () => {
+      const { user } = render(<Select items={ITEMS} />)
+
+      const input = await screen.findByRole("combobox")
+      expect(input).toBeInTheDocument()
+
+      await user.click(input)
+
+      const option1 = await screen.findByRole("option", { name: /option1/i })
+      expect(option1).toBeVisible()
+
+      await user.click(option1)
+      expect(input).toHaveTextContent(/option1/i)
+    })
+
+    test("after selected option, make sure option selection state is correct", async () => {
+      const { user } = render(
+        <MultiSelect>
+          <Option value="One">One</Option>
+          <Option></Option>
+        </MultiSelect>,
+      )
+
+      const input = screen.getByRole("combobox")
+      expect(input).toBeInTheDocument()
+      await user.click(input)
+
+      const optionList = await screen.findAllByRole("option")
+      const option1 = optionList[0]
+      const option2 = optionList[1]
+
+      expect(option1).toBeVisible()
+      expect(option2).toBeVisible()
+
+      await user.click(option1)
+      await user.click(option2)
+
+      await user.click(document.body)
+      await user.click(input)
+
+      expect(option1).toHaveAttribute("aria-checked", "true")
+      expect(option2).toHaveAttribute("aria-checked", "true")
+    })
+  })
```

Comment:
I did not see much need for this test.

Reply:
Thanks for the review!
Are you saying that I don't need to implement this test for line 261?
If so, what approach would you 108yen take to increase test coverage for use-select.ts?

Reply:
These features have not been tested.
I am sure there are many others.

- [set-maximum-number-of-selectable-values](https://yamada-ui.com/components/forms/multi-select#set-maximum-number-of-selectable-values)
- [close-dropdown-list-on-selection](https://yamada-ui.com/components/forms/multi-select#close-dropdown-list-on-selection)

Reply:
Understood.
Surely I should have created a test that tests the behavior of each feature,
not one that tests for coverage!